pro/cypress/e2e/connection/unsuccessfulLoginForUnknownUserTest.cy.ts

```diff
@@ -0,0 +1,23 @@
+import { CONSTANTS } from '../../support/constants'
+
+const connectionLinkExpected = '/connexion'
+
+describe('page connection', () => {
+  it('verify unsuccessful login for unknown user', () => {
+    cy.visit(CONSTANTS.connectionLink)
+
+    cy.get(CONSTANTS.emailId).type(CONSTANTS.emailTestData)
+
+    cy.get(CONSTANTS.passwordId).type(CONSTANTS.passwordTestData)
+
+    cy.get('button[type=submit]').click()
+
+    cy.get(CONSTANTS.emailErrorId).should('exist')
```

Comment:
J'ai l'impression que ce test passe toujours : les id recherchées sont présentent dans la page même avant d'avoir cliqué sur le bouton submit. Il faudrait plutôt vérifier que l'élément est actif : .should('be.visible')

Par ailleurs, il faudrait aussi vérifier le contenu du message d'erreur pour moi.

Reply:
Oui, dans ce cas, il vaut mieux utiliser 'should('be.visible')'.
Ainsi, la vérification sur le contenu est appliquée